### PR TITLE
links: bump version

### PR DESCRIPTION
**Pull request by kenmays**

### CODE REVIEW

File: www-client/links/patches/links-2.28.patchset [hunk 1]
```diff
@@ -27,5 +27,35 @@ index 8031d8f..f21b8f2 100644
...
  	case B_MOUSE_WHEEL_CHANGED:
 -- 
-2.28.0
+2.37.3
+
+
+From 7ecbf6ae4d023e655449c0596e644a29da12e6e7 Mon Sep 17 00:00:00 2001
+From: Ken Mays <kmays2000@gmail.com>
+Date: Mon, 12 Dec 2022 04:12:05 +1000
+Subject: Disabled ClipBoard Text for Haiku
```
Comment (waddlesplash) [inline www-client/links/patches/links-2.28.patchset:36]: Why?

Comment (kenmays) [reply www-client/links/patches/links-2.28.patchset:36]: Need to provide updated patch for upstream fix.

Comment (waddlesplash) [reply www-client/links/patches/links-2.28.patchset:36]: That doesn't answer my question.

Comment (kenmays) [reply www-client/links/patches/links-2.28.patchset:36]: My question is - is the current clipboard set/get code used within haiku.cpp 'currently' correct? Seems to break the compiler build on nightly - so I currently disabled it until verification.

Comment (kenmays) [reply www-client/links/patches/links-2.28.patchset:36]: https://github.com/haikuports/haikuports/issues/7555

Comment (waddlesplash) [reply www-client/links/patches/links-2.28.patchset:36]: What is the error?

Comment (kenmays) [reply www-client/links/patches/links-2.28.patchset:36]: Updated in ticket.

**Pull request merged**